app/components/chart/basics.js

```diff
@@ -1,41 +1,29 @@
-
-/*
- * == BSD2 LICENSE ==
```

Comment:
Is there a reason we dropped the license here?

Reply:
No strong reason, other than it seemed to be excessive to me to have it at the top of each and every file.  I asked Howard a while back if there was a regulatory reason for us having it at the top of each file, and he said that having a single LICENSE file per repo is sufficient.

I've generally stopped putting this in newly added files, and figured I'd strip it out as I migrated the files over to ES6.

I'll look into this, though.  Perhaps including it is still a good practice for some reason I'm not aware of.  What's your preference?

Reply:
It's always seemed a bit overkill to have it in every file to me. I don't know that I have a strong preference either way, it just stood out to me in the diff for this file. I'd generally support moving to a single `LICENSE` file.

Reply:
Cool.  I don't feel strongly either, but my slight preference is to not have them.  I'll leave this as-is, and update the copyright date in the `LICENCE` file